### PR TITLE
Add federation API page

### LINKED ISSUE #1709: Add HTTP API documentation for federation/v1.Replicator
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [x] New content (guide wanted)
- [ ] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

Add documentation describing the `federation/v1.Replicator` HTTP API.

Reference:
https://github.com/sensu/sensu-enterprise-go/pull/635
https://github.com/sensu/sensu-enterprise-go/issues/621
https://github.com/sensu/sensu-enterprise-go/issues/619

### LINKED ISSUE #1787: Add HTTP API documentation for federation/v1.Cluster
## This is an issue with:

- [ ] Bug (site functionality or styling)
- [ ] Errata (fix needed for doc content)
- [x] New content (guide wanted)
- [ ] Update (Add missing or refresh existing content)
- [ ] Enhancement (add new site functionality)

Add documentation describing the `federation/v1.Cluster` HTTP API.

Reference:
https://github.com/sensu/sensu-enterprise-go/issues/669
https://github.com/sensu/sensu-enterprise-go/pull/686

### LINKED ISSUE #1826: Add reference docs for etcd replication
Add reference documentation for etcd replication, and the `federation/v1.EtcdReplicator` data type. The documentation should explain how etcd replication works, and how to set up replication for RBAC resources.

### PR DESCRIPTION
**Pull request by hillaryfraley**
## Description
Adds HTTP API documentation for federation/v1

## Motivation and Context
Closes #1709
Closes https://github.com/sensu/sensu-docs/issues/1787
Closes https://github.com/sensu/sensu-docs/issues/1826

### CODE REVIEW

File: content/sensu-go/5.14/api/federation.md [hunk 1]
```diff
@@ -0,0 +1,408 @@
...
+]
+{{< /highlight >}}
+
+#### API Specification {#replicators-get-specification}
+
+/replicators (GET)  | 
+---------------|------
+description    | Returns a list of replicators.
+example url    | http://hostname:8080/api/federation/v1/replicators
+pagination     | This endpoint supports pagination using the `limit` and `continue` query parameters. See the [API overview](../overview#pagination) for details.
```
Comment (hillaryfraley) [inline content/sensu-go/5.14/api/federation.md]: Need to confirm this is true (supports pagination)

File: content/sensu-go/5.14/api/federation.md [hunk 2]
```diff
@@ -0,0 +1,408 @@
...
+
+The `/replicators` API endpoint provides HTTP GET access to a list of replicators.
+
+#### EXAMPLE {#replicator-get-example}
+
+The following example demonstrates a request to the `/replicators` API, resulting in a list of replicators.
+
+{{< highlight shell >}}
+curl http://127.0.0.1:8080/api/federation/v1/replicators -H "Authorization: Bearer $SENSU_TOKEN"
+[
```
Comment (hillaryfraley) [inline content/sensu-go/5.14/api/federation.md]: Need output/result examples for all endpoints

**Comment by hillaryfraley** [comment]
Question (for Eng?) -- which Sensu Go version does this belong with? I have it marked for 5.13, but I want to confirm that's correct.
Eric confirmed this is 5.15.

File: content/sensu-go/5.14/api/federation.md [hunk 3]
```diff
@@ -0,0 +1,408 @@
...
+title: "Federation API"
+description: "The federation API controls federation of Sensu clusters. Here’s a reference for the federation API in Sensu Go, including examples. Read on for the full reference."
+version: "5.14"
+product: "Sensu Go"
+menu:
+  sensu-go-5.14:
+    parent: api
+---
+
+- [The `/replicators` API endpoint](#the-replicators-api-endpoint)
```
Comment (echlebek) [inline content/sensu-go/5.14/api/federation.md]: I've re-named this API to `etcd-replicators`, sorry about that. The current replicators we have are specific to etcd, and I suspect we might add different ones in the future, so I'd like to disambiguate them before we ship.

File: content/sensu-go/5.14/api/federation.md (hunk 1 above)
Comment (echlebek) [reply content/sensu-go/5.14/api/federation.md]: No use case for pagination here, let's not mention it, even though I believe it's technically supported.

File: content/sensu-go/5.14/api/federation.md (hunk 2 above)
Comment (echlebek) [reply content/sensu-go/5.14/api/federation.md]: Here's an example of an EtcdReplicator resource:
```
{
  "api_version": "federation/v1",
  "type": "EtcdReplicator",
  "metadata": {
    "name": "my_replicator"
  },
  "spec": {
    "insecure": true,
    "url": "http://127.0.0.1:3379",
    "api_version": "core/v2",
    "resource": "CheckConfig",
    "replication_interval_seconds": 5
  }
}
```

File: content/sensu-go/5.14/api/federation.md (hunk 1 above)
Comment (hillaryfraley) [reply content/sensu-go/5.14/api/federation.md]: Thank you! I'll delete it.

Comment (hillaryfraley) [inline content/sensu-go/5.14/api/federation.md]:
```suggestion
```

File: content/sensu-go/5.14/api/federation.md (hunk 2 above)
Comment (hillaryfraley) [reply content/sensu-go/5.14/api/federation.md]: Thank you. Is it OK to use this as the output example for all of the endpoints?

File: content/sensu-go/5.15/api/federation.md [hunk 4]
```diff
@@ -0,0 +1,326 @@
...
+  }
+]
+{{< /highlight >}}
+
+## The `/cluster/:cluster` endpoint {#the-clustercluster-endpoint}
+
+_**NOTE**: Only cluster admins have PUT and DELETE access to clusters._
+
+### `/cluster/:cluster` (PUT) {#clustercluster-put}
+
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md:389]: It looks like we're missing a description and example for the PUT endpoint.

File: content/sensu-go/5.15/api/federation.md [hunk 5]
```diff
@@ -0,0 +1,326 @@
...
+    },
+    "spec": {
+      "api_urls": {
+        "url": "http://10.0.0.1:8080",
+        "url": "http://10.0.0.2:8080",
+        "url": "http://10.0.0.3:8080"
+      }
+    }
+  }
+]
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```
[
    {
        "type": "Cluster",
        "api_version": "federation/v1",
        "metadata": {
            "name": "my_cluster"
        },
        "spec": {
            "api_urls": [
                "http://10.0.0.1:8080",
                "http://10.0.0.2:8080",
                "http://10.0.0.3:8080"
            ]
        }
    }
]
```

File: content/sensu-go/5.15/api/federation.md [hunk 6]
```diff
@@ -0,0 +1,326 @@
...
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```
[
    {
        "type": "Cluster",
        "api_version": "federation/v1",
        "metadata": {
            "name": "my_cluster"
        },
        "spec": {
            "api_urls": [
                "http://10.0.0.1:8080",
                "http://10.0.0.2:8080",
                "http://10.0.0.3:8080"
            ]
        }
    }
]
```

File: content/sensu-go/5.15/api/federation.md [hunk 7]
```diff
@@ -0,0 +1,326 @@
...
+      "name": "my_cluster"
+    },
+    "spec": {
+      "api_urls": {
+        "url": "http://10.0.0.1:8080",
+        "url": "http://10.0.0.2:8080",
+        "url": "http://10.0.0.3:8080"
+      }
+    }
+  }
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```
{
    "type": "Cluster",
    "api_version": "federation/v1",
    "metadata": {
        "name": "my_cluster"
    },
    "spec": {
        "api_urls": [
            "http://10.0.0.1:8080",
            "http://10.0.0.2:8080",
            "http://10.0.0.3:8080"
        ]
    }
}
```

**Review by nikkictl — CHANGES_REQUESTED** [review]
Aside from the other comments, all API routes here (etcd-replicator and cluster) should be prefixed with `/api/enterprise/federation/v1/`. I believe `enterprise` is missing from the paths on this page.

File: content/sensu-go/5.15/api/federation.md (hunk 4 above)
Comment (hillaryfraley) [reply content/sensu-go/5.15/api/federation.md:389]: Yeah -- I don't have an example for either of the PUT endpoints for federation (etcd-replicators or clusters)

**Comment by hillaryfraley** [comment]
I added PUT examples, but they are just guesses.

File: content/sensu-go/5.15/api/federation.md [hunk 8]
```diff
@@ -0,0 +1,372 @@
...
+[
+  {
+    "api_version": "federation/v1",
+    "type": "replicator",
+    "metadata": {
+      "name": "my_replicator"
+    },
+    "spec": {
+      "insecure": true,
+      "url": "http://127.0.0.1:3379",
```
Comment (echlebek) [inline content/sensu-go/5.15/api/federation.md]: Let's use 
```
"url": "http://remote-etcd.example.com:2379"
```
here and in the other examples

File: content/sensu-go/5.15/api/federation.md [hunk 9]
```diff
@@ -0,0 +1,372 @@
...
+
+## The `/cluster` endpoint
+
+### `/cluster` (GET)
+
+The `/cluster` endpoint provides HTTP GET access to a list of clusters.
+
+#### EXAMPLE {#cluster-get-example}
+
+The following example demonstrates a request to the `/cluster` endpoint, resulting in a list of clusters.
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
The following example demonstrates a request to the `/clusters` endpoint, resulting in a list of clusters.
```

File: content/sensu-go/5.15/api/federation.md [hunk 10]
```diff
@@ -0,0 +1,372 @@
...
+#### API Specification {#etcd-replicatorsetcd-replicator-delete-specification}
+
+/etcd-replicators/:etcd-replicator (DELETE) | 
+--------------------------|------
+description               | Deletes the specified replicator from Sensu.
+example url               | http://hostname:8080/api/enterprise/federation/v1/etcd-replicators/{etcd-replicator_name}
+response codes            | <ul><li>**Success**: 204 (No Content)</li><li>**Missing**: 404 (Not Found)</li><li>**Error**: 500 (Internal Server Error)</li></ul>
+
+
+## The `/cluster` endpoint
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
## The `/clusters` endpoint
```

File: content/sensu-go/5.15/api/federation.md [hunk 11]
```diff
@@ -0,0 +1,372 @@
...
+
+### `/cluster` (GET)
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
### `/clusters` (GET)
```

File: content/sensu-go/5.15/api/federation.md [hunk 12]
```diff
@@ -0,0 +1,372 @@
...
+
+The `/cluster` endpoint provides HTTP GET access to a list of clusters.
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
The `/clusters` endpoint provides HTTP GET access to a list of clusters.
```

File: content/sensu-go/5.15/api/federation.md (hunk 9 above)
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
The following example demonstrates a request to the `/clusters` endpoint, resulting in a list of clusters.
```

File: content/sensu-go/5.15/api/federation.md [hunk 13]
```diff
@@ -0,0 +1,372 @@
...
+
+#### EXAMPLE {#cluster-get-example}
+
+The following example demonstrates a request to the `/cluster` endpoint, resulting in a list of clusters.
+
+{{< highlight shell >}}
+curl http://127.0.0.1:8080/api/enterprise/federation/v1/cluster -H "Authorization: Bearer $SENSU_TOKEN"
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
curl http://127.0.0.1:8080/api/enterprise/federation/v1/clusters -H "Authorization: Bearer $SENSU_TOKEN"
```

File: content/sensu-go/5.15/api/federation.md [hunk 14]
```diff
@@ -0,0 +1,372 @@
...
+                "http://10.0.0.3:8080"
+            ]
+        }
+    }
+]
+{{< /highlight >}}
+
+#### API Specification {#cluster-get-specification}
+
+/cluster (GET)  |
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
/clusters (GET)  | 
```

File: content/sensu-go/5.15/api/federation.md [hunk 15]
```diff
@@ -0,0 +1,372 @@
...
+    }
+]
+{{< /highlight >}}
+
+#### API Specification {#cluster-get-specification}
+
+/cluster (GET)  | 
+---------------|------
+description    | Returns a list of clusters.
+example url    | http://hostname:8080/api/enterprise/federation/v1/cluster
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
example url    | http://hostname:8080/api/enterprise/federation/v1/clusters
```

File: content/sensu-go/5.15/api/federation.md [hunk 16]
```diff
@@ -0,0 +1,372 @@
...
+                "http://10.0.0.1:8080",
+                "http://10.0.0.2:8080",
+                "http://10.0.0.3:8080"
+            ]
+        }
+    }
+]
+{{< /highlight >}}
+
+## The `/cluster/:cluster` endpoint {#the-clustercluster-endpoint}
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
## The `/clusters/:cluster` endpoint {#the-clustercluster-endpoint}
```

File: content/sensu-go/5.15/api/federation.md [hunk 17]
```diff
@@ -0,0 +1,372 @@
...
+
+_**NOTE**: Only cluster admins have PUT and DELETE access to clusters._
+
+### `/cluster/:cluster` (PUT) {#clustercluster-put}
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
### `/clusters/:cluster` (PUT) {#clustercluster-put}
```

File: content/sensu-go/5.15/api/federation.md [hunk 18]
```diff
@@ -0,0 +1,372 @@
...
+        "name": "my_cluster"
+    },
+    "spec": {
+        "api_urls": [
+            "http://10.0.0.1:8080",
+            "http://10.0.0.2:8080",
+            "http://10.0.0.3:8080"
+        ]
+    }
+}
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]: This should probably have shell highlighting around it.

Comment (nikkictl) [reply content/sensu-go/5.15/api/federation.md]: Screen Shot 2019-10-22 at 1 54 13 PM

File: content/sensu-go/5.15/api/federation.md [hunk 19]
```diff
@@ -0,0 +1,372 @@
...
+
+#### API Specification {#clustercluster-put-specification}
+
+/cluster/:cluster (PUT) |
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
/clusters/:cluster (PUT) | 
```

File: content/sensu-go/5.15/api/federation.md [hunk 20]
```diff
@@ -0,0 +1,372 @@
...
+        ]
+    }
+}
+
+#### API Specification {#clustercluster-put-specification}
+
+/cluster/:cluster (PUT) | 
+----------------|------
+description     | Creates or updates the specified cluster.
+example URL     | http://hostname:8080/api/enterprise/federation/v1/cluster/{cluster_name}
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
example URL     | http://hostname:8080/api/enterprise/federation/v1/clusters/{cluster_name}
```

File: content/sensu-go/5.15/api/federation.md [hunk 21]
```diff
@@ -0,0 +1,372 @@
...
+            "http://10.0.0.1:8080",
+            "http://10.0.0.2:8080",
+            "http://10.0.0.3:8080"
+        ]
+    }
+}
+{{< /highlight >}}
+response codes  | <ul><li>**Success**: 201 (Created)</li><li>**Malformed**: 400 (Bad Request)</li><li>**Error**: 500 (Internal Server Error)</li></ul>
+
+### `/cluster/:cluster` (DELETE) {#cluster-delete}
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
### `/clusters/:cluster` (DELETE) {#cluster-delete}
```

File: content/sensu-go/5.15/api/federation.md [hunk 22]
```diff
@@ -0,0 +1,372 @@
...
+_**NOTE**: Only cluster admins have PUT and DELETE access to clusters._
+
+### `/cluster/:cluster` (PUT) {#clustercluster-put}
+
+#### EXAMPLE {#clustercluster-put-example}
+
+The following example demonstrates a request to update the cluster `my_cluster`.
+
+curl -X PUT \
+http://127.0.0.1:8080/api/enterprise/federation/v1/cluster/my_cluster -H "Authorization: Bearer $SENSU_TOKEN" \
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
http://127.0.0.1:8080/api/enterprise/federation/v1/clusters/my_cluster -H "Authorization: Bearer $SENSU_TOKEN" \
```

File: content/sensu-go/5.15/api/federation.md [hunk 23]
```diff
@@ -0,0 +1,372 @@
...
+            "http://10.0.0.3:8080"
+        ]
+    }
+}
+{{< /highlight >}}
+response codes  | <ul><li>**Success**: 201 (Created)</li><li>**Malformed**: 400 (Bad Request)</li><li>**Error**: 500 (Internal Server Error)</li></ul>
+
+### `/cluster/:cluster` (DELETE) {#cluster-delete}
+
+The `/cluster/:cluster` endpoint provides HTTP DELETE access to delete the specified cluster from Sensu.
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
The `/clusters/:cluster` endpoint provides HTTP DELETE access to delete the specified cluster from Sensu.
```

File: content/sensu-go/5.15/api/federation.md [hunk 24]
```diff
@@ -0,0 +1,372 @@
...
+
+### EXAMPLE {#clustercluster-delete-example}
+
+The following example shows a request to delete the cluster `my_cluster`, resulting in a successful HTTP 204 No Content response.
+
+{{< highlight shell >}}
+curl -X DELETE \
+-H "Authorization: Bearer $SENSU_TOKEN" \
+http://127.0.0.1:8080/api/enterprise/federation/v1/cluster/my_cluster
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
http://127.0.0.1:8080/api/enterprise/federation/v1/clusters/my_cluster
```

File: content/sensu-go/5.15/api/federation.md [hunk 25]
```diff
@@ -0,0 +1,372 @@
...
+
+HTTP/1.1 204 No Content
+{{< /highlight >}}
+
+#### API Specification {#etcd-replicatorsetcd-replicator-delete-specification}
+
+/cluster/:cluster (DELETE) | 
+--------------------------|------
+description               | Deletes the specified cluster from Sensu.
+example url               | http://hostname:8080/api/enterprise/federation/v1/cluster/{cluster_name}
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
example url               | http://hostname:8080/api/enterprise/federation/v1/clusters/{cluster_name}
```

**Review by nikkictl — CHANGES_REQUESTED** [review]
Some more feedback on the API url is included as well as some formatting. As for the example, it might be more helpful for users if the cluster was named something more meaningful so it relates to a real world application (ex. `us-west-2a`).

File: content/sensu-go/5.15/api/federation.md [hunk 26]
```diff
@@ -0,0 +1,374 @@
...
+
+{{< highlight shell >}}
+curl http://127.0.0.1:8080/api/enterprise/federation/v1/clusters -H "Authorization: Bearer $SENSU_TOKEN"
+[
+    {
+        "type": "Cluster",
+        "api_version": "federation/v1",
+        "metadata": {
+            "name": "us-west-2a"
+            },
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]: The indent here is a little off, it should align with metadata.

File: content/sensu-go/5.15/api/federation.md [hunk 27]
```diff
@@ -0,0 +1,374 @@
...
+response type  | Array
+response codes | <ul><li>**Success**: 200 (OK)</li><li>**Error**: 500 (Internal Server Error)</li></ul>
+output         | {{< highlight shell >}}
+[
+    {
+        "type": "Cluster",
+        "api_version": "federation/v1",
+        "metadata": {
+            "name": "us-west-2a"
+            },
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]: Same with this indent.

File: content/sensu-go/5.15/api/federation.md [hunk 28]
```diff
@@ -0,0 +1,374 @@
...
+_**NOTE**: Only cluster admins have PUT and DELETE access to clusters._
+
+### `/clusters/:cluster` (PUT) {#clusterscluster-put}
+
+#### EXAMPLE {#clusterscluster-put-example}
+
+The following example demonstrates a request to update the cluster `us-west-2a`.
+
+{{< highlight shell >}}
+curl -X PUT \
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]: See an example [PUT](https://docs.sensu.io/sensu-go/5.14/api/checks/#checkscheck-put) for the correct arguments, and where to pass the request body.

**Review by nikkictl — CHANGES_REQUESTED** [review]
We're so close!

File: content/sensu-go/5.15/api/federation.md [hunk 29]
```diff
@@ -0,0 +1,381 @@
...
+
+/clusters/:cluster (PUT) | 
+----------------|------
+description     | Creates or updates the specified cluster.
+example URL     | http://hostname:8080/api/enterprise/federation/v1/clusters/us-west-2a
+payload         | {{< highlight shell >}}
+{
+    "type": "Cluster",
+    "api_version": "federation/v1",
+    "metadata": {},
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]: This renders incorrectly due to a bug in our code (see https://github.com/sensu/sensu-enterprise-go/issues/704). Let's display the resource as it should be printed.
```suggestion
    "metadata": {
            "name": "us-west-2a"
      },
```

The metadata should be

File: content/sensu-go/5.15/api/federation.md [hunk 30]
```diff
@@ -0,0 +1,381 @@
...
+    "spec": {
+        "metadata": {
+            "name": "us-west-2a"
+        },
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]: We should remove this metadata (see previous comment).

File: content/sensu-go/5.15/api/federation.md [hunk 31]
```diff
@@ -0,0 +1,383 @@
...
+example URL     | http://hostname:8080/api/enterprise/federation/v1/clusters/us-west-2a
+payload         | {{< highlight shell >}}
+{
+    "type": "Cluster",
+    "api_version": "federation/v1",
+    "metadata": {
+            "name": "us-west-2a"
+      },
+    "spec": {
+        "metadata": {
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
```

File: content/sensu-go/5.15/api/federation.md [hunk 32]
```diff
@@ -0,0 +1,383 @@
...
+            "name": "us-west-2a"
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
```

File: content/sensu-go/5.15/api/federation.md [hunk 33]
```diff
@@ -0,0 +1,381 @@
...
+example URL     | http://hostname:8080/api/enterprise/federation/v1/clusters/us-west-2a
+payload         | {{< highlight shell >}}
+{
+    "type": "Cluster",
+    "api_version": "federation/v1",
+    "metadata": {
+            "name": "us-west-2a"
+      },
+    "spec": {
+        },
```
Comment (hillaryfraley) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
```

File: content/sensu-go/5.15/api/federation.md (hunk 30 above)
Comment (hillaryfraley) [reply content/sensu-go/5.15/api/federation.md]: I was wondering what that was doing there. thank you

File: content/sensu-go/5.15/api/federation.md [hunk 34]
```diff
@@ -0,0 +1,380 @@
...
+----------------|------
+description     | Creates or updates the specified cluster.
+example URL     | http://hostname:8080/api/enterprise/federation/v1/clusters/us-west-2a
+payload         | {{< highlight shell >}}
+{
+    "type": "Cluster",
+    "api_version": "federation/v1",
+    "metadata": {
+            "name": "us-west-2a"
+      },
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]:
```suggestion
    },
```

File: content/sensu-go/5.15/api/federation.md [hunk 35]
```diff
@@ -0,0 +1,380 @@
...
+- [The `/etcd-replicators` endpoint](#the-etcd-replicators-endpoint)
+	- [`/etcd-replicators` (GET)](#etcd-replicators-get)
+	- [`/etcd-replicators` (POST)](#etcd-replicators-post)
+- [The `/etcd-replicators/:etcd-replicator` endpoint](#the-etcd-replicatorsetcd-replicator-endpoint)
+	- [`/etcd-replicators/:etcd-replicator` (GET)](#etcd-replicatorsetcd-replicator-get)
+	- [`/etcd-replicators/:etcd-replicator` (PUT)](#etcd-replicatorsetcd-replicator-put)
+ 	- [`/etcd-replicators/:etcd-replicator` (DELETE)](#etcd-replicatorsetcd-replicator-delete)
+- [The `/clusters` endpoint](#the-clusters-endpoint)
+ 	- [`/clusters` (GET)](#clusters-get)
+- [The `/clusters/:cluster` endpoint](#the-clusterscluster-endpoint)
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md:20]: Looks like we're missing a section to GET a specific cluster.
```
- [`/clusters/:cluster` (GET)](#clusterscluster-get)
```

File: content/sensu-go/5.15/api/federation.md [hunk 36]
```diff
@@ -0,0 +1,442 @@
...
+  }
+}
+{{< /highlight >}}
+
+#### API Specification {#etcd-replicatorsetcd-replicator-get-specification}
+
+/clusters/:cluster (GET) | 
+---------------------|------
+description          | Returns the specified cluster.
+example url          | http://hostname:8080/api/enterprise/federation/v1/clusters/{cluster_name}
```
Comment (nikkictl) [inline content/sensu-go/5.15/api/federation.md]: In the other endpoints, you used the literal `us-west-2a` name. I would stay consistent here.
```suggestion
example url          | http://hostname:8080/api/enterprise/federation/v1/clusters/us-west-2a
```

**Review by nikkictl — APPROVED** [review]
Just one last comment and I think the `/cluster` API docs are good to go! I'll leave it to @echlebek to OK the `/etcd-replicators` API.

**Comment by hillaryfraley** [comment]
Thank you for all of your help this week, @nikkixdev !

File: content/sensu-go/5.14/api/federation.md (hunk 2 above)
Comment (echlebek) [reply content/sensu-go/5.14/api/federation.md]: Oops, I guess I never responded to this. Let's sync up on slack about this.

**Review by echlebek — APPROVED** [review]
This looks good to me!

**Pull request merged**